src/main/java/com/KacFlor/ShopSpring/controller/UserController.java

```diff
@@ -14,16 +15,27 @@
 @RequestMapping(
         path = {"user"}
 )
-public class UserController {
+public class UserController{
 
     private final UserService userService;
 
     @Autowired
-    public UserController(UserService userService) {
+    public UserController(UserService userService){
         this.userService = userService;
     }
 
     @GetMapping
-    public List<User> getUser() {
-        return this.userService.getUser();}
+    public List<User> getUser(){
+        return this.userService.getUser();
+    }
+
+    @GetMapping("user")
+    public String helloUser(){
+        return "Hello User";
+    }
+
+    @GetMapping("admin")
```

Comment:
This endpoint url is "user/admin" here, this is as test right?

Reply:
done

Reply:
I expected yes or no answer